deploy/roles/python-build/tasks/main.yml

```diff
@@ -0,0 +1,36 @@
+# main.yml - Installs Python 2.7 from custom RPM
+---
+- name: Make sure git is installed
+  sudo: yes
+  yum:
+    name: git
+    state: latest
+
+- name: Clone git repo
+  sudo: no
+  git:
+    dest: "~/python27"
+    repo: https://github.com/cfpb/python27-for-el6.git
+
+- name: Run bootstrap script
+  sudo: no
+  command: "sh ./bootstrap.sh"
+  args:
+    chdir: "~/python27"
+
+- name: Build python RPM
```

Comment:
This was partly for convenience, although I do see having Python 2.7 installed as a dependency of the platform overall.  If we consider it a dependency, then we either have to install it as part of the build or require users to install it manually first (or to provide the RPM manually in some way).  By allowing the build to happen within the provisioning process, we make it easier for the user to use the platform with minimal manual configuration.  There is an option to use a custom repository instead, which is a way of avoiding the build if you want, and I think that's sufficient flexibility.  I'm not sure where else I would put it, that wouldn't just make things more complex unnecessarily.  Open to suggestions though.